src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Names/Names.ts

```diff
@@ -18,6 +18,6 @@ export class Service {
         } else {
             ret = "TR__RESOURCE";
         }
-        return amount !== 1 ? ret + "_PLURAL" : ret;
+        return amount != 1 ? ret + "_PLURAL" : ret;
```

Comment:
While this fixes the symptom, it also hides the underlying issue. My interpretation is the following:

`amount`was not `1`, but `"1"`. This is because of chain of unexpected behavior:

-  The backend returns all values as strings.
-  The frontend converts the values automatically based on information from the meta API
-  The meta API does not contains "search" fields, e.g. `SIPool.count`
-  `SIPool.count` is used as `amount` in the listing

So the proper fix would be to convert the value to number in Listing.ts